lib/cli.js

```diff
@@ -259,6 +268,8 @@ const meowOptions = {
 			type: 'boolean',
 		},
 		config: {
+			// TODO
+			// alias: 'c',
```

Comment:
[suggest] Let's open an issue instead of leaving a comment:

```suggestion
```

Reply:
Could you catch this up? Or, you can just remove the change.

Reply:
Ill remove it but won't _resolve_ until someone creates an issue.

Reply:
If needed, please open an issue. At least, the change is unrelated to this pull request.

Reply:
#6690